LBHackney-IT/lbh-fss-portal-frontend
# add validation and tests to AddUser component

**Description**

Added validation and tests to AddUser component

**Test instructions**

- navigate to **/users/add**, check form validation works by putting incorrect values in fields (e.g. blank name, blank email, email address with incorrect pattern, name too long, email too long) 
- check tests in **AddUser.test.js** passes


**Ticket**

https://app.clickup.com/t/9e5u93
